salesforce/10.10/modules/ROOT/pages/salesforce-connector-studio.adoc

```diff
@@ -72,367 +74,242 @@ Subscribe to a topic. This source provides topic-related notification settings f
+|===
 
-Important fields include the following:
+[[oauth-jwt]]
+=== OAuth JWT Authentication
 
-* *Consumer key* (required): +
-Consumer key for the Salesforce-connected app. See <<Create a Consumer Key>>.
-* *Consumer secret* (required) +
-Consumer secret for the connector to access Salesforce +
-* *Listener config* (required) +
-Configuration for the listener, for example, `HTTP_Listener_config` +
-* *Callback path* (required) +
-Path for the callback, for example, `/callback`
-* *Authorize path* (required) +
-Path for authorization, for example, `/authorize` +
-* *External callback url* +
-Callback URL, for example, `+http://localhost:8085/callback+`
+OAuth JWT uses OAuth 2.0 with a JSON Web token (JWT) request for user authentication. Important fields include the following:
 
-=== OAuth JWT
+[%header,cols="30s,70a"]
+|===
+|Field a|Description
+|Consumer key | Consumer key for the Salesforce-connected app. See the Salesforce documentation for information about generating a consumer key.
+| Key store | Path to the keystore that signs data during authentication. See <<Generate a Keystore File>>.
+| Store password | Password for the keystore
+| Password | Corresponding password
+| Certificate Alias| Alias of the certificate to be used if the specified keystore contains multiple certificates
+| Principal | Username of the Salesforce user
+|===
 
-OAuth JWT uses OAuth 2.0 with a JSON Web token (JWT) request for user authentication.
+[[oauth-saml]]
+=== OAuth SAML Authentication
 
-Important fields include the following:
+OAuth SAML uses OAuth 2.0 with a signed SAML 2.0 assertion to request an access token. Important fields include the following:
 
-* *Consumer key* (required)
-+
-Consumer key for the Salesforce-connected app. See <<Create a Consumer Key>>.
-+
-* *Key store* (required)
-+
-See <<Generate a Keystore File>>.
-+
-* *Store password* (required)
-+
-Password for the keystore
-+
-* *Certificate Alias* (optional)
-+
-The alias of the certificate to be used if the specified keystore contains multiple certificates
-+
-* *Principal* (required)
-+
-Username of the Salesforce user
+[%header,cols="30s,70a"]
+|===
+|Field a|Description
+|Consumer key | Consumer key for the Salesforce-connected app. See the Salesforce documentation for information about generating a consumer key.
+| Key store | Path to the keystore that signs data during authentication. See <<Generate a Keystore File>>.
+| Store password | Password for the keystore
+| Password | Corresponding password
+| Certificate Alias| Alias of the certificate to be used if the specified keystore contains multiple certificates
+| Principal | Username of the Salesforce user
+|===
 
-=== OAuth SAML
+[[oauth-username-password]]
+=== OAuth Username Password Authentication
 
-OAuth SAML uses OAuth 2.0 with a signed SAML 2.0 assertion to request an access token.
+OAuth Username Password uses OAuth 2.0 with a user and password for authentication. Important fields include the following:
```

Comment:
Reworded just because in the case of "Username Password," I read "password uses...." Your decision. If we change this one, maybe we should the others as well.
```suggestion
OAuth Username Password authentication uses OAuth 2.0 with a user and password. Important fields include the following:
```